### ISSUE TITLE
Support for ReactiveCollection.AddRangeOnScheduler

### ISSUE THREAD

**Issue by BADF00D** [runceel/ReactiveProperty#10]
Hi,
I would like to have the possibility add an range of items to a ReactiveCollection. 

What do you think?

**Comment by runceel** [comment]
Hi. Thank you for your feedback.

I think so too.I add AddRangeOnScheduler method to a ReactiveCollection. And release to nuget.
Please confirm.

**Comment by BADF00D** [comment]
Hi,
I just downloaded version 2.2.6, but the AddRangeOnScheduler seems to be missing. I can see that the method  has been added in commit 6b59946779144b68d8d9056f54fb3dda9f60c1c8 within github. Maybe something went wrong during deployment of the package?

Beste Greetings: 
![reactiveproperty missing addrange](https://cloud.githubusercontent.com/assets/2852160/10428204/1fa3386c-70f0-11e5-9a0a-afc70ff82b5d.PNG)
